api/api.csproj

```diff
@@ -22,4 +22,40 @@
     <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0" />
     <DotNetCliToolReference Include="Microsoft.DotNet.Watcher.Tools" Version="2.0.0" />
   </ItemGroup>
+  
+  
+  <Target Name="DebugRunWebpack" BeforeTargets="Build" Condition=" '$(Configuration)' == 'Debug' ">
+	<Message Importance="high" Text="Installing any non-peer missing dependencies.." />
+    <Exec Command="npm install" />
+	
+	<!-- It would seem, that the npm install cmd targets the parent directory(natural path of the routine), and yet, the node webpack execution does not. So we go up one level  -->
+    <Message Importance="high" Text="Performing first-run Webpack build..." />
+    <Exec Command="node ../node_modules/webpack/bin/webpack.js --config ../client-react/webpack.config.js" />
```

Comment:
Can you explain the need for this?  The webpack middleware should handle running webpack on-demand as assets are requested.  It is configured here: https://github.com/bradymholt/aspnet-core-react-template/blob/master/api/Startup.cs#L92

Reply:
This is purely for the visual studio launch scenario. It targets the folder of the application by default (./api/) which means node_modules is one up. One could go into tools, options, Web Packages, and edit the items there to put a "../node_modules" resolution path, but figured this would prevent that installation step. My main concern with it though is whether or not it executes on VS Code (untested by me). 
Without it, visual studio tries to run webpack from wrong folder  (can't resolve webpack in bin of node_modules).
Definitely open to suggestions on this aspect